gatewayconfig/bluetooth/characteristics/diagnostics_characteristic.py

```diff
@@ -43,7 +43,7 @@ def get_p2pstatus(self):
         logger.debug('Diagnostics miner_bus')
         miner_bus = dbus.SystemBus()
         logger.debug('Diagnostics miner_object')
-        miner_object = miner_bus.get_object('com.helium.Miner', '/')
+        miner_object = miner_bus.get_object('com.helium.Miner', '/', introspect=False)
```

Comment:
Nice! We can further investigate in https://github.com/NebraLtd/hm-diag/issues/53 what was causing these errors in the first place and if it is something we should be concerned about.